roles/pg_replicant/tasks/main.yml

```diff
@@ -0,0 +1,27 @@
+# based on https://github.com/lesovsky/ansible-postgresql-sr-on-el6
+
+- name: stop postgresql on replicant
+  service:
+    name: postgresql
+    state: stopped
+
+- name: remove old postgresql cluster on replicant
+  file:
+    path: XXX
+    state: absent
+
+# pg_basebackup
+
+- name: configure recovery.conf
+  template:
+    src: XXX/recovery.conf
+    dest: "{{ postgresql_data_dir }}/recovery.conf"
```

Comment:
We should probably keep with our existing path scheme rather than introduce the `postgresql_data_dir` variable, no?

It could possibly become a default in the role. Maybe a role default of `postgresql_data_dir: /var/lib/postgresql/{{ postgres_version }}/main/`?